generators/quarkus/entity-files.js

```diff
@@ -44,7 +44,7 @@ export const entityQuarkusFiles = {
         }),
         javaMainPackageTemplatesBlock({
             condition: generator => generator.service === 'serviceClass' && !generator.embedded,
-            renameTo: (data, filename) => filename.replace('ServiceImpl.java', 'Service.java'),
+            renameTo: (_data, file) => file.replace('service/impl', 'service').replace('Impl.java', '.java'),
             templates: ['_entityPackage_/service/impl/_entityClass_ServiceImpl.java'],
```

Comment:
It seems easier to rename the template. Why not choose that route?

Reply:
The same template is used for serviceClass and serviceImpl. So a rename is necessary one way or another.

It's also implemented in the base JHipster like that.